alacritty.yml

```diff
@@ -275,6 +275,10 @@ selection:
 
 hide_cursor_when_typing: false
 
+# Start a thread to monitor configuration file changes.
+# (changes require restart)
+start_config_monitor_thread: true
```

Comment:
Can we name this `live_config_reload` instead of `start_config_monitor_thread`? I think there's no reason to mention how live reloading is implemented.